Add bedrock/signals accelerator (#1680)
snowplow/documentation: tutorials/signals-agentic-accelerator/conclusion.md

---
title: "Conclusion and next steps"
sidebar_label: "Conclusion"
position: 6
description: "Summary of the Signals-powered AI agent accelerator, cleanup instructions, and next steps for production deployment."
keywords: ["conclusion", "cleanup", "next steps", "AgentCore", "Signals", "production"]
date: "2026-03-27"
---

> You have built an AI agent that combines real-time behavioral data from Snowplow Signals with persistent memory from AWS Bedrock AgentCore Memory. The agent can:

Reviewer: ```suggestion
You've built an AI agent that combines real-time behavioral data from Snowplow Signals with persistent memory from AWS Bedrock AgentCore Memory. The agent can:
```

* Answer queries using custom tools (destination lookup, experience info, web search)
* Fetch behavioral attributes from Signals to understand what the user is doing right now
* Retrieve and store customer context across conversations using AgentCore Memory
* Deliver personalized responses that reflect both browsing behavior and historical preferences

> This pattern - tools + behavioral context + persistent memory - applies to any customer-facing agent, from support bots to shopping assistants to content recommenders.

Reviewer: ```suggestion
This pattern - tools plus behavioral context plus persistent memory - is useful in any customer-facing agent, from support bots to shopping assistants to content recommenders.
```

If you connected the [SEA Explorer demo app](/tutorials/signals-agentic-accelerator/connect-demo-app), you also have a working frontend that calls Bedrock with Signals and Memory integration, demonstrating the full end-to-end flow.

## Clean up

Delete the AgentCore Memory resources created during this accelerator to avoid ongoing costs.

```python
from bedrock_agentcore.memory import MemoryClient
from boto3.session import Session

boto_session = Session()
region = boto_session.region_name

memory_client = MemoryClient(region_name=region)
memory_client.gmcp_client.delete_memory(memoryId=memory_id)
print(f"Deleted memory resource: {memory_id}")
```

> If you published a Signals service and attribute group, you can remove them from your Sandbox or CDI environment through the Signals dashboard.

Reviewer: add another optional cell at the end of the notebook that deletes them? bit of a faff to have to go hunting for them in Console
